DependencyInjection/MonologExtension.php

```diff
@@ -75,6 +75,9 @@ public function load(array $configs, ContainerBuilder $container)
             $handlers = [];
 
             foreach ($config['handlers'] as $name => $handler) {
+                if(false === $handler['enabled']) {
```

Comment:
CS is borken
and if this is a boolean, there's no need for any comparisons:
```suggestion
                if (!$handler['enabled']) {
```